src/App.js

```diff
@@ -1,33 +1,63 @@
-import { useState } from 'react';
-import Login from './components/Login';
-import SignUp from './components/SignUp';
-import { Button, Container } from './components/styles';
+import { useState } from "react";
+import Login from "./components/Login";
+import SignUp from "./components/SignUp";
+import { Button, Container } from "./components/styles";
+import Home from "./components/Home"
 
+import {
+  Routes,
+  Route,
+} from "react-router-dom";
+import CreateBlog from './components/CreateBlog';
 
 function App() {
- const [showModal, setShowModal] = useState(false)
- const [showSignModal, setShowSignModal] = useState(false)
+  const [showLoginModal, setShowLoginModal] = useState(false);
+  const [showSignModal, setShowSignModal] = useState(false);
 
- const openModal = () => {
-    setShowModal(prev => !prev);
+  const [isauthenticated, setIsAuthenticated] = useState(false);
+
+  const openLoginModal = () => {
+    setShowLoginModal((prev) => !prev);
```

Comment:
Why not just explicitly `setShowLoginModal` to `true`?

Reply:
i was trying something new out